### ISSUE TITLE
Can you add W25Q256JVEQ?  

### ISSUE THREAD

**Issue by bananamarcia** [nofeletru/UsbAsp-flash#43]
![50be992e-8938-43d7-94ea-7b5f1339d8ac](https://user-images.githubusercontent.com/93555338/139749852-376bc99d-346e-4292-b214-de721bb86a94.jpg)

**Comment by nofeletru** [comment]
Download new chiplist.xml
https://github.com/nofeletru/UsbAsp-flash/blob/master/chiplist.xml

**Comment by bananamarcia** [comment]
There are two W25Q256JVEQ with two diffrent id... can you pls tell me which is mine?

**Comment by nofeletru** [comment]
> There are two W25Q256JVEQ with two diffrent id... can you pls tell me which is mine?

W25Q256JV IQI /IN /JQ 184019h 
W25Q256JV M * /JM * 187019h 
it does not matter  since it uses same commands

**Comment by bananamarcia** [comment]
okay, thanks...

**Comment by bananamarcia** [comment]
could you help me with another matter??
I have an HP Elitebook 830 g5, and my ch341a programmer doesn't read my bios W25Q256JVEQ... WHAT SHOULD I DO?

**Comment by nofeletru** [comment]
Did you desolder chip? If program do not  read chip ID then this some wireing problem.

**Comment by bananamarcia** [comment]
I didn't desolder the chip. 
I will try to explain the situation, maybe maybe I was wrong in some steps. I just disconnected the battery and the CMOS from my hp elitebook 830 g5. Then without desolding the BIOS chip I connected it to the programmer with the tweezers provided in the kit, after I installed the programmer drivers.
You said it could be a wiring problem, about what, the programmer or the tweezers?
